eks-distro-base/Makefile

```diff
@@ -136,6 +140,20 @@ PYTHON_VARIANT_NAME=eks-distro-minimal-base-python
 BASE_PYTHON_VARIANT_VERSIONS=3.9
 # ************************************************************
 
+
+# ****************** GOLANG VARIANTS *************************
+GOLANG_VARIANT_NAME=eks-distro-minimal-base-golang
+BASE_GOLANG_VARIANT_VERSIONS=1.15 1.16 1.17 1.18 1.19
+
+GOLANG_1.15_FULL_VERSION=1.15.15-1
+GOLANG_1.16_FULL_VERSION=1.16.15-1
+GOLANG_1.17_FULL_VERSION=1.17.13-1
+GOLANG_1.18_FULL_VERSION=1.18.7-1
+GOLANG_1.19_FULL_VERSION=1.19.2-1
```

Comment:
Not sure if it matters, but the latest are 1.18.8 and 1.19.3

Reply:
I went with what we have the builder base currently. I figure we'll keep these two in sync